src/VAP/BGSObjectDetector/BGSObjectDetector.csproj

```diff
@@ -1,16 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Library</OutputType>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <ApplicationIcon />
-    <StartupObject />
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="OpenCvSharp4" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.ubuntu.18.04-x64" Version="4.0.0.20190108" />
-    <PackageReference Include="OpenCvSharp4.runtime.win" Version="4.0.0.20190108" />
```

Comment:
📝 These are only needed for executables. Removing the references from libraries simplifies version management a bit.